src/modules/vtol_att_control/standard.cpp

```diff
@@ -363,6 +363,13 @@ void Standard::update_mc_state()
 		return;
 	}
 
+	// Do not engage pusher assist during a failsafe event
+	// There could be a problem with the fixed wing drive
+	if (_attc->get_vtol_vehicle_status()->vtol_transition_failsafe) {
+		_pusher_throttle = 0.0f;
```

Comment:
Was it it necessary to zero pusher_throttle here?

It's set to 0 in update_vtol_state() -
 https://github.com/PX4/Firmware/blob/master/src/modules/vtol_att_control/standard.cpp#L182
Then again here for some reason - https://github.com/PX4/Firmware/blob/master/src/modules/vtol_att_control/standard.cpp#L370